### DISCUSSION TITLE
umi4 proxy配置失效

### DISCUSSION THREAD

**Discussion by woshiji1987** [umijs/umi#8449] [Q&A]
umi4 proxy配置失效，proxy配置本身和umi3时候保持一致
![DD1803A9-0A97-4135-B559-DF69A884E3E0](https://user-images.githubusercontent.com/33452948/177699251-bae2ee2e-d379-4dcc-9938-c9bd9baf4ff8.png)
![770648D3-B1D6-4EBA-9F63-D5790DDBE2D4](https://user-images.githubusercontent.com/33452948/177699264-36f2eba7-edda-4f20-93fe-dca6e1281fa1.png)

**Comment by fz6m** [comment]
给一个复现 demo 看看。

**Comment by huarse** [comment]
遇到了类似的问题，不过只在项目里面有问题，用 https://httpbin.org/post 测试又是OK的，现在还在想办法找原因
